userguide/dailyuse/anbox.rst

```diff
@@ -1,13 +1,15 @@
 Android apps
 ========================
 
-`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer that allows you to run Android apps on GNU/Linux operating systems such as UBports.
+`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as UBports.
 
 .. note::
-    When "host" is used in this document, it refers to another device which you can connect your Ubuntu Touch device to. Your host device must have ``adb`` and ``fastboot`` installed.
+    "Computer" refers to another device you can connect your Ubuntu Touch device to.
+    Your attached computer must have ``adb`` and ``fastboot`` installed.
```

Comment:
```suggestion
    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
```
I argue the connection happens via USB, and that the USB (cable) is attached.
In any event it is easier to understand with more explanations(?)